ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -1359,6 +1438,9 @@ func (r *RayClusterReconciler) updateHeadInfo(ctx context.Context, instance *ray
 	if headPod != nil {
 		instance.Status.Head.PodIP = headPod.Status.PodIP
 		instance.Status.Head.PodName = headPod.Name
+	} else {
+		instance.Status.Head.PodIP = ""
+		instance.Status.Head.PodName = ""
```

Comment:
Clean the Head Pod information from the `Status.Head` when there is no Head Pod, ex: cluster suspended.